EIPS/eip-multi_party_contract.md

```diff
@@ -0,0 +1,320 @@
+---
+eip: 3742
+title: Multi Party Contract Standard
+author: Saurabh Santhosh (@saurabhsanthosh)
+type: Standards Track
+category: ERC
+status: Draft
+created: 2021-08-15
+---
+
+
+## Simple Summary
```

Comment:
We recently deprecated the "Simple Summary" in favor of a description field in the preamble. Please see the [EIP template ](https://github.com/ethereum/EIPs/blob/master/eip-template.md)for an example.